hera_mc/data/HERA_350.txt

```diff
@@ -318,31 +318,31 @@ HH8	541018.40	6601070.74	1052.64
 HH9	541033.00	6601070.74	1052.52
 HH10	541047.60	6601070.74	1052.61
 HH22	541062.20	6601079.17	1052.45
-HA320	540762.90	6601530.14	1048.22
-HA321	540908.90	6601530.14	1048.44
-HA322	541054.90	6601530.14	1048.79
-HA323	541200.90	6601530.14	1049.28
-HA324	540689.90	6601412.13	1049.05
+HB320	540762.90	6601530.14	1048.22
```

Comment:
What is the significance of switching from HA to HB here? I don't actually know what either mean.